x/concentrated-liquidity/swaps_test.go

```diff
@@ -0,0 +1,162 @@
+package concentrated_liquidity_test
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/osmosis-labs/osmosis/v13/x/concentrated-liquidity/internal/math"
+	"github.com/osmosis-labs/osmosis/v13/x/gamm/types"
+	swaproutertypes "github.com/osmosis-labs/osmosis/v13/x/swaprouter/types"
+)
+
+var _ = suite.TestingSuite(nil)
+
+func (s *KeeperTestSuite) TestCLPoolSimpleSwapExactAmountIn() {
+	currPrice := sdk.NewDec(5000)
+	currSqrtPrice, err := currPrice.ApproxSqrt() // 70.710678118654752440
+	s.Require().NoError(err)
+	currTick := math.PriceToTick(currPrice) // 85176
+	lowerPrice := sdk.NewDec(4545)
+	lowerTick := math.PriceToTick(lowerPrice) // 84222
+	upperPrice := sdk.NewDec(5500)
+	upperTick := math.PriceToTick(upperPrice) // 86129
+
+	defaultAmt0 := sdk.NewInt(1000000)
+	defaultAmt1 := sdk.NewInt(5000000000)
+
+	swapFee := sdk.ZeroDec()
+
+	type param struct {
+		tokenIn           sdk.Coin
+		tokenOutDenom     string
+		tokenOutMinAmount sdk.Int
+		expectedTokenOut  sdk.Int
+	}
+
+	tests := []struct {
+		name      string
+		param     param
+		expectErr bool
+	}{
+		{
+			name: "Proper swap foo > bar",
+			param: param{
+				tokenIn:           sdk.NewCoin("foo", sdk.NewInt(42000000)),
```

Comment:
Is there a Wolfram link you are using to QA this? How can I be sure that the values are correct here?

Reply:
"trust me bro"

jk, I am using the same base test cases from pool_test, I can copy over the wolfram links from there though

Reply:
Yeah I think duplicating a link would be helpful. While I assumed that there is probably another test case using the same values, it is just much more easier having it available in this test directly